packages/types/src/signUp.ts

```diff
@@ -133,6 +133,11 @@ export type PrepareVerificationParams =
       strategy: SamlStrategy | EnterpriseSSOStrategy;
       redirectUrl?: string;
       actionCompleteRedirectUrl?: string;
+    }
+  | {
+      strategy: EnterpriseSSOStrategy;
+      redirectUrl?: string;
+      actionCompleteRedirectUrl?: string;
```

Comment:
I have to ask now that i see it twice, but why not this ?
```suggestion
      strategy: SamlStrategy | EnterpriseSSOStrategy 
| EnterpriseSSOStrategy
      redirectUrl?: string;
      actionCompleteRedirectUrl?: string;
    }
```